projects/nitrious_engine/pressure_ladder.py

```diff
@@ -1,60 +1,35 @@
 from CoolProp.CoolProp import PropsSI
```

Comment:
Is this file suppose to be in this?

Reply:
mmmm no that's a good point. Lemme change that

Reply:
I mean I did it on purpose but I think there is a better place for that